src/coreclr/vm/ecalllist.h

```diff
@@ -161,6 +161,7 @@ FCFuncStart(gCOMTypeHandleFuncs)
     FCFuncElement("IsValueType", RuntimeTypeHandle::IsValueType)
     FCFuncElement("IsInterface", RuntimeTypeHandle::IsInterface)
     FCFuncElement("IsByRefLike", RuntimeTypeHandle::IsByRefLike)
+    FCFuncElement("_IsUnmanagedFunctionPointer", RuntimeTypeHandle::IsUnmanagedFunctionPointer)
```

Comment:
Instead of the `_` prefix, I'd go with the `Internal` suffix. It is the more common idiom for these types of calls.